cadence/README.md

```diff
@@ -216,6 +216,37 @@ assert_eq!(
 );
 ```
 
+### Value Packing
+
+Value packing is supported for `HISTOGRAM`, `DISTRIBUTION`, and `TIMING` metrics for Datadog agent
```

Comment:
nit: Could you change the phrasing here a bit? 
```suggestion
Value packing is supported for histograms, distributions, and timers when writing to the Datadog agent
```

Reply:
Done in 014e678!